src/error.js

```diff
@@ -109,19 +109,16 @@ export function reportError(error, opt_associatedElement) {
 
   // Report to console.
   if (self.console) {
+    const output = (console.error || console.log);
     if (error.messageArray) {
-      (console.error || console.log).apply(console,
-          error.messageArray);
+      output.apply(console, error.messageArray);
     } else {
       if (element) {
-        (console.error || console.log).call(console,
-            element.tagName.toLowerCase() +
-                (element.id ? ' with id ' + element.id : '') + ':',
-            error.message);
+        output.call(console, error.message, element);
```

Comment:
I'll explain my original thinking behind the old state. Lets discuss that, but I tend to agree that your PR makes things better:

I felt people might be confused that the DOM, especially when printed to the console looks different from their raw HTML -> Because we add all kinds of classes and attributes to it, add child elements, etc.

Having said that, this is probably better. Certainly in your case, where the elements aren't always an AmpElement and mutation from original source is expected.

Could you add a screenshot of the Chrome console to the PR description?